src/engraving/dom/rest.cpp

```diff
@@ -486,17 +486,18 @@ int Rest::computeVoiceOffset(int lines, LayoutData* ldata) const
 
 int Rest::computeWholeRestOffset(int voiceOffset, int lines) const
```

Comment:
In my version of this PR, I changed the name from `computeWholeRestOffset` to `computeWholeOrBreveRestOffset`, to reflect what it's new function is.